packages/cdktf/lib/terraform-data-source.ts

```diff
@@ -12,6 +12,8 @@ import { keysToSnakeCase, deepMerge } from "./util";
 import { ITerraformDependable } from "./terraform-dependable";
 import { ref, dependable } from "./tfExpression";
 import { IInterpolatingParent } from "./terraform-addressable";
+import { IIterator } from "./iterator";
+import assert = require("assert");
```

Comment:
Why are we not using the normal import from syntax here?

Reply:
The normal import failed `yarn watch`, this one didn't 😅 (We're also already using this weird syntax for other third party deps in cdktf)